core/common/buffer.cpp

```diff
@@ -3,18 +3,18 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <utility>
+#include <boost/algorithm/hex.hpp>
 
```

Comment:
No need for the empty space between non-std-includes

Reply:
I noticed that order of includes is wrong here, but blank line should be present between std and non std includes according to [that](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)